libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/dialogs/ConfirmationDialog.kt

```diff
@@ -48,9 +48,9 @@ fun ConfirmationDialog(
     onThirdButtonClicked: () -> Unit = {},
     onDismiss: () -> Unit = {},
     shape: Shape = AlertDialogDefaults.shape,
-    containerColor: Color = MaterialTheme.colorScheme.surfaceVariant,
-    iconContentColor: Color = MaterialTheme.colorScheme.onSurfaceVariant,
-    titleContentColor: Color = MaterialTheme.colorScheme.onSurfaceVariant,
+    containerColor: Color = MaterialTheme.colorScheme.surface,
```

Comment:
Why not using `AlertDialogDefaults.containerColor` (this is the default for `AlertDialog`). I am not 100% sure, but since now we stick to `material3`, it would have more sense to keep default behavior.

Reply:
Done in [`67374f8`](https://github.com/vector-im/element-x-android/pull/174/commits/67374f8fcbf9509aa8d671a1cd0f6989d7133ea6).